recipes/av2/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "av2" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/argoai/argoverse2-api/archive/refs/tags/v{{ version }}.tar.gz
```

Comment:
Is there any reason you're using PyPI as a source? GitHub is fine as well but it's normally larger so we try to save CI resources by using PyPI. If you cannot change it – no problem though.

Reply:
Yeah, it might be best to hold off on this for the time being if possible. There's a slight discrepancy in the versioning on PyPI and the most recent tagged release on Github. I should be able to fix on the next release, though.